src/script/components/MessagesList/Message/ContentMessage/ContentMessage.styles.ts

```diff
@@ -19,7 +19,7 @@
 
 import {CSSObject} from '@emotion/react';
 
-export const messageBodyWrapper = (isEphemeralMessage: boolean): CSSObject => ({
+export const messageBodyWrapper = (isEphemeralMessage: boolean = false): CSSObject => ({
```

Comment:
You don't need to pass type if You add default state ;)

```suggestion
export const messageBodyWrapper = (isEphemeralMessage = false): CSSObject => ({
```